site/content/en/docs/drivers/qemu.md

```diff
@@ -21,6 +21,14 @@ To start minikube with the qemu driver:
 minikube start --driver=qemu
 ```
 
+## Special features
+
+minikube start supports some qemu specific flags:
+
+* **`--qemu-firmware-path`**: The path to the firmware image to be used.
+  * Note: while the flag should override the config, if the flag does not take effect try running `minikube delete --all --purge`.
+  * Macports: if you are installing [minikube](https://ports.macports.org/port/minikube/) and [qemu](https://ports.macports.org/port/qemu/) via Macports on a Mac with M1, use the following flag: `--qemu-firmware-path=/opt/local/share/qemu/edk2-aarch64-code.fd`
```

Comment:
```suggestion
  * MacPorts: if you are installing [minikube](https://ports.macports.org/port/minikube/) and [qemu](https://ports.macports.org/port/qemu/) via MacPorts on a Mac with M1, use the following flag: `--qemu-firmware-path=/opt/local/share/qemu/edk2-aarch64-code.fd`
```

Reply:
The website https://www.macports.org/ uses MacPorts, does it have to be Macports?

Reply:
I'm recommending it to be `MacPorts`, it's currently `Macports` in the PR

Reply:
Nice catch, `MacPorts` is my preference as well.